docs: update readme docs (#5740)
PreMiD/Presences: .github/CONTRIBUTING.md

> # Contrubutions

Reviewer: nice
```suggestion
# Contributions
```

Author: Forgot to push this change lmfao

As an open-sourced project, we welcome contributions with open arms. 

> > You should ensure you read the entirety of this doucment before contributing to the repository. We require the following rules for good practice. 

Reviewer 2: ```suggestion
> You should ensure you read the entirety of this document before contributing to the repository. We require the following rules for good practice. 
```
Typo

> > You should ensure you read the entirety of this document before contributing to the repository. We require the following rules for good practice. 

Reviewer: why is this a quote? Also the phrasing of this is very weird, specifically in the "you should ensure" and "We require the following rules" parts

## Issues 

> Issues are designed for tracking bugs, feature requests and service requets by using their associated templates. The content of each field should be detiailed thoroughly before being submitted. 

Reviewer: an issue is not a contribution, this file should be about making presences, testing them and opening PRs

Author: Yes it is

Reviewer: No it isn’t.

Author: Github class it as a contribution
https://github.com/actions/runner/blob/main/docs/contribute.md

Issues not relavent to the repository or low quality issues are invalid and therefore will be closed promptly.

## Pull Requests

> We use pull requests to maintain repository standards. Anybody may open a pull request provided their changes follow our [Guidelines](https://docs.premid.app/en/dev/presence/guidelines)

Reviewer: It's good to link the guidelines, but this needs to be MUCH more detailed in order to explain how to load a local presence and everything you need to do before opening a PR. I recommend splitting it into two parts: "Creating a presence" and "Updating a presence"

Author: That should be on the docs

Reviewer: One doesn't have to go through the docs to find basic info like that. That should be explained here like it is on every other good repo

Author: Do you want the guidelines on the repo too then?

Reviewer: Documentation explaining everything needed to make a PR (stuff mentioned above), yes

### Commits
The following set of rules apply for pull requests & commits to the repository:

>  - **Use commitlint**. For more information regarding this, please see [COMMIT_CONVENTION.md](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

Reviewer: ```suggestion
 - **Use commitlint**. For more information regarding this, please see [Git Commit Message Convention](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)
```

Author: Then the sentence doesn't make sense

Reviewer: ok then add a "the" before

>  - **Use the imperative tone** - "chore: update assets" > "chore: updates

Reviewer 2: ```suggestion
 - **Use the imperative tone** - "chore: update assets" > "chore: updates"
```
Should this be like so? 👀

>  - **Use the imperative tone** - "chore: update assets" > "chore: updates"

Reviewer: nothing about this is using the imperative *tone*

Author: better
```suggestion
 - **Use the imperative tone** - "chore: update assets" > "chore: updates assets"
```

Reviewer: `updates` is still not imperative

Author: Give a better example then

>  - **Use the present tense** - "feat: add feature" > "feat: added feature"

Reviewer: this commit pains me
```suggestion
 - **Use the present tense** - "feat: add support for X" > "feat: added support for X"
```

Author: I don't care, it is just a really basic template

Reviewer: having feat and feature is not good commit practice. Just apply it
